src/storage/gc_worker.rs

```diff
@@ -211,26 +246,97 @@ impl<E: Engine> GCRunner<E> {
         );
         Ok(())
     }
-}
 
-impl<E: Engine> Runnable<GCTask> for GCRunner<E> {
-    fn run(&mut self, mut task: GCTask) {
+    fn destroy_range(&self, _: &Context, start_key: &Key, end_key: &Key) -> Result<()> {
+        debug!(
+            "destroying range start_key: {}, end_key: {}",
+            start_key, end_key
+        );
+
+        // TODO: Refine usage of errors
+
+        let local_storage = self.local_storage.as_ref().ok_or_else(|| {
+            let e: Error = box_err!("destroy range not supported: local_storage not set");
+            warn!("destroy range failed: {:?}", &e);
+            e
+        })?;
+
+        // Convert keys to RocksDB layer form
+        // TODO: Logic coupled with raftstore's implementation. Maybe better design is to do it in
```

Comment:
What does this todo mean?

Reply:
This part of code's correctness, to some degree, depends on whether raftstore's implementation is changed. So I think it's not very elegant to put code like `keys::data_key` here.